docs/accessibility/accessibility-module.md

```diff
@@ -1,10 +1,14 @@
 Accessibility module
 ===
 
-The accessibility of Highcharts is critical to us. Every Highcharts license includes our Accessibility module, which helps ensure your charts are as accessible as possible. We use the WCAG 2.1 standard as our guideline for this, as well as involving users with disabilities in our testing and feature development.
+The accessibility of Highcharts is critical to us. Accessibility should be fundamental to any software project, not an additional add on. Every Highcharts license includes our Accessibility module, which helps ensure your charts are as accessible as possible.  We use the WCAG 2.1 standard as our guideline for this, as well as involving users with disabilities in our testing and feature development.
```

Comment:
Maybe replace "not an additional add on" with e.g. "not an afterthought". Also looks like an extra space before "We use the WCAG 2.1 standard...".